### ISSUE TITLE
Build failing for rust 1.60

### ISSUE THREAD

**Issue by ericmarkmartin** [de-vri-es/jws#3]
I'm unable to build the crate---see the following output from `cargo build`

```
❯ cargo build
   Compiling jws v0.2.6 (/home/ericmarkmartin/Development/jws)
error[E0277]: the trait bound `M: InnerInit` is not satisfied
   --> src/hmac.rs:119:16
    |
119 |     let mut mac = M::new_from_slice(key).unwrap();
    |                   ^^^^^^^^^^^^^^^^^ the trait `InnerInit` is not implemented for `M`
    |
    = note: required because of the requirements on the impl of `KeyInit` for `M`
note: required by a bound in `hmac::Mac::new_from_slice`
   --> /home/ericmarkmartin/.cargo/registry/src/github.com-1ecc6299db9ec823/digest-0.10.3/src/mac.rs:35:15
    |
35  |         Self: KeyInit;
    |               ^^^^^^^ required by this bound in `hmac::Mac::new_from_slice`
help: consider further restricting this bound
    |
118 | fn compute_mac<M: Mac + hmac::digest::InnerInit>(encoded_header: &[u8], encoded_payload: &[u8], key: &[u8]) -> hmac::digest::CtOutput<M> {
    |                       +++++++++++++++++++++++++

error[E0277]: the trait bound `M: InnerInit` is not satisfied
   --> src/hmac.rs:126:16
    |
126 |     let mut mac = M::new_from_slice(key).unwrap();
    |                   ^^^^^^^^^^^^^^^^^ the trait `InnerInit` is not implemented for `M`
    |
    = note: required because of the requirements on the impl of `KeyInit` for `M`
note: required by a bound in `hmac::Mac::new_from_slice`
   --> /home/ericmarkmartin/.cargo/registry/src/github.com-1ecc6299db9ec823/digest-0.10.3/src/mac.rs:35:15
    |
35  |         Self: KeyInit;
    |               ^^^^^^^ required by this bound in `hmac::Mac::new_from_slice`
help: consider further restricting this bound
    |
125 | fn verify_mac<M: Mac + hmac::digest::InnerInit>(encoded_header: &[u8], encoded_payload: &[u8], signature: &[u8], key: &[u8]) -> Result<()> {
    |                      +++++++++++++++++++++++++

For more information about this error, try `rustc --explain E0277`.
error: could not compile `jws` due to 2 previous errors
```

**Comment by de-vri-es** [comment]
The problem was caused by the `digest` crate, version 0.10.2 is not backwards compatible. I've fixed the breakage and released it as 0.2.7, but we'll have to keep an eye on what happens next to see if more action is required if `digest` fixes the problem.

Either way, thanks for reporting the issue!

**Comment by de-vri-es** [comment]
Versions `0.10.0` and `0.10.1` of the digest crate have been yanked, so the current fix is the proper one. No further action is needed here :)

I've also yanked `0.2.6` of `jws` since it was compatible with `digest` version `0.10.0`.

Thanks again for reporting it!
